editor/editor_node.cpp

```diff
@@ -5671,6 +5671,9 @@ EditorNode::EditorNode() {
 				if (DisplayServer::get_singleton()->screen_get_dpi(screen) >= 192 && DisplayServer::get_singleton()->screen_get_size(screen).y >= 1400) {
 					// hiDPI display.
 					scale = 2.0;
+				} else if (DisplayServer::get_singleton()->screen_get_size(screen).y >= 2000) {
+					// default to 125% on 4k displays
+					scale = 1.25;
```

Comment:
We might as well default to a scale of 150%. It'll be more usable out of the box:

```suggestion
				} else if (DisplayServer::get_singleton()->screen_get_size(screen).y >= 1700) {
					// Likely a hiDPI display, but we aren't certain due to the returned DPI.
					// Use an intermediate scale to handle this situation.
					scale = 1.5;
```

Also, I decreased the Y threshold to 1700 so that 3200x1800 displays also use 150% scaling. (This display size is found on laptops such as the Dell XPS 13.)

Reply:
Wasn't sure if 125% or 150% was the better solution. On the few 4k screens I have access to both look ok.
Switched to 150% for now.